golang/thrift/server.go

```diff
@@ -61,6 +67,11 @@ func (s *Server) Register(svr TChanServer) {
 	}
 }
 
+// RegisterHealthHandler registers User health endpoint handler into TChannel.
```

Comment:
nit: User does not need to be capitalized. I'd probably make the message:
RegisterHealthHandler uses the user-specified function f for the Health endpoint.